src/components/tag/template.test.js

```diff
@@ -0,0 +1,66 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('tag')
+
+describe('tag component', () => {
+  it('renders the default example with strong tag and text correctly', () => {
+    const { $ } = render('tag', examples.default)
+
+    const $component = $('.govuk-c-tag')
+    expect($component.get(0).tagName).toEqual('strong')
+    expect($component.text()).toContain('alpha')
+  })
+
+  it('renders classes correctly', () => {
+    const { $ } = render('tag', {
+      classes: 'govuk-c-tag--inactive',
+      text: 'alpha'
+    })
+
+    const $component = $('.govuk-c-tag')
+    expect($component.hasClass('govuk-c-tag--inactive')).toBeTruthy()
+  })
+
+  it('renders custom text correctly', () => {
+    const { $ } = render('tag', {
+      text: 'alpha'
```

Comment:
This is the same text as in the default example, so we could change the logic to hard code `alpha` and I think all of these tests would still pass? Suggest specifying some text that would not normally appear in any of our examples.

Reply:
Done!